### DISCUSSION TITLE
Table of Contents

### DISCUSSION THREAD

**Discussion by arielvieira** [sergiodxa/remix-utils#200] [Ideas]
Hi, first of all I want to say thanks for the library, it's amazing and had save me so many times.

But as someone who use it a lot, I find myself going back a lot in the readme to find the docs for any specific util, so that maybe a good idea to add a `table of contents` in the root readme so we can easily skim though and find the util we are looking for.

I'm happy to make a PR if applicable.

**Comment by sergiodxa** [comment]
You can use the GitHub built-in table of content for Markdown files feature https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

**Reply by arielvieira** [reply]
Didn't knew that existed, thanks for the letting me know
